### ISSUE TITLE
CIFAR-100 dataset in train_cifar.py has only 20 classes.

### ISSUE THREAD

**Issue by takedarts** [YimianDai/open-aff#14]
I am interested in AFF-ResNets, and I evaluated the performance of AFF-ResNets with both this implementation and my own implementation.
Through the experiments, I found a bug.

In `train_cifar.py`, CIFAR-100 dataset is loaded as follows:

```
train_data = gluon.data.DataLoader(
   gluon.data.vision.CIFAR100(train=True).transform_first(transform_train),
   batch_size=batch_size, shuffle=True, last_batch='discard', num_workers=num_workers)
```

but, according to API reference of gluon, CIFAR-100 with default settings has only 20 classes.

https://mxnet.apache.org/versions/1.7/api/python/docs/api/gluon/data/vision/datasets/index.html#mxnet.gluon.data.vision.datasets.CIFAR100

Option `fine_label=True` is required to compare the performance with other models on CIFAR-100 classification task.

**Comment by YimianDai** [comment]
My god... Many thanks for this important information. It is a terrible mistake. I didn't notice this `fine_label=True` thing. 

I will soon delete my thing on paperswithcode and modify the paper. 

Many thanks for your information.

**Comment by takedarts** [comment]
It's my pleasure if I can help with your work.
I am looking forward to new results.

**Comment by YimianDai** [comment]
Many thanks!

I will try to report the new CIFAR-100 results training from scratch in 24 hours. 

Luckily, the ablation study results still hold since they are all compared in 20 superclasses (I will explain it in the paper that the accuracies are on 20 superclasses), and the ImageNet experiments has no such problems.

**Comment by YimianDai** [comment]
Emmm... I have submitted the Machine Request to the HPC for nearly 24 hours, but it is still in queue... 

I will keep updating the progress.

**Comment by YimianDai** [comment]
Hi. The accuracy of AFF-ResNeXt-47 training from scratch on the fine-grained CIFAR-100  is 85.4%.  The trained models and log file is uploaded.

**Comment by takedarts** [comment]
I trained ResNet-32 (not AFF-ResNet-32) in your implementation with `fine_label=True` on my computer with a 1080Ti GPU.
The validation accuracy is 82.6% which is close to accuracy of WideResNet-34k4 in my own implementation.

Your AFF-ResNeXt-47 result (85.4%) also seems to be reasonable.
